doc/CodingStandard.md

```diff
@@ -433,6 +433,37 @@ if ((logLevelValue >= 0) && (logLevelValue <= 4)) {
 * C order of operation rules are complicated; avoid confusion and mistakes by using parentheses to make the order explicit.
 * Combining assignment and tests violates the "Try to do only one thing on each line of code" principle.
 
+### Constructor member-initializer lists
+* Each member should be initialized on a separate line.
+* Each member initialization should be preceded by the colon or comma and a space.
+* Each member initialization should be indented at least one tab.
+* The `:` and `,` should be aligned.
+
+Correct
+```c
+MyClass(uint32_t foo, uint32_t bar, uint32_t baz)
+	: foo(foo)
+	, bar(bar)
+	, baz(baz)
+{ ... }
```

Comment:
Perhaps the examples (both good and bad) should have the braces on separate lines (as they normally would):
```c
{
    ...
}
```